api/v1/models/masquerading.go

```diff
@@ -34,8 +34,12 @@ type Masquerading struct {
 	// Enum: [BPF iptables]
 	Mode string `json:"mode,omitempty"`
 
-	// Any packet sent to IP addr belonging to CIDR will not be SNAT'd
-	SnatExclusionCidr string `json:"snat-exclusion-cidr,omitempty"`
+	// Any packet sent to IPv4 addr belonging to CIDR will not be SNAT'd
+	SnatExclusionCidrV4 string `json:"snat-exclusion-cidr-v4,omitempty"`
```

Comment:
Nit: `SNATExclusionCIDRv4`

```
SNATExclusionCIDRv4 exempts SNAT from being performed on any packet sent to an IPv4 address that belongs inside this CIDR.
```

Reply:
This is an autogenerated field using openapi spec. I don't think it's possible for us to change the name of the variable right? :confused:

Reply:
Yes, this whole file should be autogenerated and should not be modified manually. :)